### ISSUE TITLE
URL parameters seem to keep Annotation Cards from opening

### ISSUE THREAD

**Issue by mkdir-washington-edu** [hypothesis/support#106]
**Describe the bug**
Students in the LMS app were intermittently having trouble annotating an assignment made using https://via.hypothes.is/https://www.usgs.gov/special-topic/water-science-school/science/groundwater-what-groundwater?amp=&qt-science_center_objects=0#qt-science_center_objects. While some students could annotate, others did not get an open Card after selecting "Annotate" in the Adder.

**To Reproduce**
1. Go to https://via.hypothes.is/https://www.usgs.gov/special-topic/water-science-school/science/groundwater-what-groundwater?amp=&qt-science_center_objects=0#qt-science_center_objects
2. Select text and select "Annotate" in the Adder
3. If an annotation Card opens, you may need to refresh and try again until you see the error.

**Expected behavior**
The Card should open, and it does, sometimes. Other times, a new card will not open, and there will be a Console error (see below). The Console error does **not** appear if the Card is opening.

**Screenshots**
![image](https://user-images.githubusercontent.com/22603275/80001179-13c22580-848c-11ea-9de8-30519615e6df.png)

**Desktop (please complete the following information):**
 - OS: Mac Mojave 10
 - Browser: Chrome 81

**Additional context**
https://hypothesis.zendesk.com/agent/tickets/9646

**Comment by klemay** [comment]
Adding link to Slack discussion in case it's helpful: https://hypothes-is.slack.com/archives/C2BLQDKHA/p1587565091274500

**Comment by LMS007** [comment]
@mkdir-washington-edu was this screen shot from your repro steps or the user?

I suspect that whatever error the user saw may be different than this one.  The only way to arrive at that exception is if the parent frames don't exist (e.g., you don't run it inside of an LMS) and the `ancestorLevel` was passed in as a number. We set that as "2" for everything.

So its quit possible the sidebar broke for another reason, but it would be very helpful to see the output in chrome in the moddle app itself or at least get a screen shot on the console at that time

**Comment by mkdir-washington-edu** [comment]
@LMS007 That screenshot was from my repo steps. No screenshots from the user.

**Comment by mkdir-washington-edu** [comment]
From the Teacher: 

I used a google search to find the website. I opened the article and copied the URL from the browser bar. This is the link I would have copied. 

https://www.usgs.gov/special-topic/water-science-school/science/groundwater-what-groundwater?qt-science_center_objects=0#qt-science_center_objects 

In hypothes,is, I create an assignment. Click on the assignment then click on "add URL..." Then I pasted the website in.
